contentcuration/contentcuration/frontend/shared/views/MarkdownEditor/plugins/image-upload/image-md-to-html.js

```diff
@@ -7,37 +7,19 @@
  *
  * will be converted to
  *
- * "<img src='path/to/checksum.ext' alt='alt-text' width="100" height="200">"
+ * "<span is='markdown-image-field'>![alt-text](${placeholer}/checksum.ext =100x200)</span>"
  *
  */
-import { IMAGE_PLACEHOLDER, CLASS_IMG_FIELD } from '../../constants';
+import { IMAGE_REGEX, imageMdToImageFieldHTML } from './index';
 
-const IMAGE_REGEX = /!\[([^\]]*)]\(([^/]+\/([^\s]+))(?:\s=([0-9.]+)x([0-9.]+))*\)/g;
+// convert markdown images to image editor field custom elements
 
 export default markdown => {
   const matches = [...markdown.matchAll(IMAGE_REGEX)];
   matches.forEach(match => {
-    const imgMarkdown = match[0];
-    const description = match[1];
-    const filePathWithPlaceholder = match[2];
-    const fileNameWithExtension = match[3];
-    const width = match[4];
-    const height = match[5];
-
-    const imagePath = `/content/storage/${fileNameWithExtension[0]}/${fileNameWithExtension[1]}`;
-    const src = filePathWithPlaceholder.replace(IMAGE_PLACEHOLDER, imagePath);
-    const checksum = fileNameWithExtension.split('.')[0];
-    markdown = markdown.replace(
-      imgMarkdown,
-      `<img
-        alt="${description}"
-        src="${src}"
-        width="${width || 'auto'}"
-        height="${height || 'auto'}"
-        class="${CLASS_IMG_FIELD}"
-        data-checksum="${checksum}"
-      />`
-    );
+    // Make sure the exercise placeholder is there
```

Comment:
I don't really follow that comment either... it's probably a leftover from some old code!

...But I've deleted all of this anyway because this code was causing the nesting custom elements bug.